pulsar-proxy/pom.xml

```diff
@@ -70,6 +70,11 @@
       <artifactId>jetty-servlets</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-proxy</artifactId>
```

Comment:
I didn't know about this. It looks a very good approach. 

Can you verify that it will internally follow HTTP redirects?

Reply:
It should, I've explicitly added the option but I will test this.

Reply:
I spent a day or two following another approach until I had enough and went searching for a better answer, this seemed to fit the bill perfectly.